bin/mail/2016-04-26-changeorg-petition/generate-lists.rb

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env ruby
+require_relative '../mailing-common/mailing-list-utils'
+
+# All US contacts
+us_contacts_query = %q(
+  !(role_s:student) && (
+    create_ip_country_s:"United States" ||
+    hoc_country_s:"us" ||
+    location_country_code_s:"US" ||
+    location_country_s:"United States"
+  )
+)
+
+fields = %w[email_s name_s]
+us_contacts = query_subscribed_contacts(q: us_contacts_query, fl:fields)
```

Comment:
This is inside a query string. Hound is being very picky, but I'll fix it anyway.